### PR TITLE
VyOS: acl model added

### PR DESCRIPTION
**Pull request by rohitthakur2590**
Signed-off-by: Rohit Thakur <rohitthakur2590@outlook.com>

### CODE REVIEW

File: models/vyos/acl/vyos_acl.yaml [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+                    address:
+                      description: Source netwrok address
+                      type: str
+                destination:
+                  description:
+                    - Specify the packet destination
+                  type: dict
+                  suboptions:
+                    type:
+                      description: Destination network address type
```
Comment (samccann) [inline models/vyos/acl/vyos_acl.yaml]:
```suggestion
                      description: Destination network address type.
```

File: models/vyos/acl/vyos_acl.yaml [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+                      type: str
+                      choices: ['any', 'host', 'inverse-mask', 'netwrok']
```
Comment (samccann) [inline models/vyos/acl/vyos_acl.yaml]:
```suggestion
                      choices: ['any', 'host', 'inverse-mask', 'network']
```

File: models/vyos/acl/vyos_acl.yaml [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+                    address:
+                      description: Destination netwrok address
```
Comment (samccann) [inline models/vyos/acl/vyos_acl.yaml]:
```suggestion
                      description: Destination network address.
```

File: models/vyos/acl/vyos_acl.yaml [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+        'status': ['preview'],
+        'supported_by': 'network'
+    }
+
+NETWORK_OS: vyos
+RESOURCE: acls
+COPYRIGHT: Copyright 2019 Red Hat
+
+DOCUMENTATION: |
+  module: vyos_acls
```
[the remark this answers is not in the document]
Comment (rohitthakur2590) [reply models/vyos/acl/vyos_acl.yaml]: As its a list , we will keep it as 'vyos_acls',  same has been discussed with Trishna as well.

**Comment by rohitthakur2590** [comment]
firewall_rules have different model

**Pull request closed without merging**